docs/en/modules/net_info.md

```diff
@@ -0,0 +1,79 @@
+# net_info Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-01-21 | [smcl](http://blog.mclemon.io/esp8266-contributing-to-the-nodemcu-ecosystem) | [wolfgangr](https://github.com/wolfgangr/nodemcu-firmware) | [net_info.c](../../../app/modules/net_info.c)
```

Comment:
I ust learned the full name of Sean McLemon when he dropped into issue #1758 .
I'd prefer to quote the full name.

>  Sean is the origin and you are contributor 

never mind.
Maybe even better than "Maintainer" - less obligations, more the one shot thing?